integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -159,12 +161,16 @@ public static void initialize(String appPropsFile) throws Exception {
           "/usr/local/packages/aime/ias/run_as_root \"chmod 777 "
               + pvRoot
               + "/acceptance_test_pv\"");
+      logger.info("Creating " + sitconfigDir);
+      TestUtils.exec("/usr/local/packages/aime/ias/run_as_root \"mkdir -p " + sitconfigDir + "\"");
+      TestUtils.exec("/usr/local/packages/aime/ias/run_as_root \"chmod 777 " + sitconfigDir + "\"");
```

Comment:
We should have no dependency on 'run_as_root' since it's only available internally - I thought @vanajamukkara has been working on factoring this out.

Reply:
@vanajamukkara can you answer this please ?

Reply:
@sankarpn why do you have to create sitconfigDir  in BaseTest.java if its only needed for ITSitConfig test? If you have to create a sub directory under RESULT_ROOT, I don't think you have to create with run_as_root.

Reply:
> 
> 
> We should have no dependency on 'run_as_root' since it's only available internally - I thought @vanajamukkara has been working on factoring this out.

We still use run_as_root to create PV_ROOT and RESULT_ROOT directories on Jenkins.

Reply:
Moved the sitconfigDir creation in ITSitConfig class.